src/components/ContentSidebar/ActivitySidebar.js

```diff
@@ -56,7 +56,7 @@ const ActivitySidebar = ({
     getAvatarUrl,
     getUserProfileUrl
 }: Props) => (
-    <SidebarContent title={<FormattedMessage {...messages.sidebarActivityTitle} />}>
+    <SidebarContent hasScrollbar={false} title={<FormattedMessage {...messages.sidebarActivityTitle} />}>
```

Comment:
Why are we setting this to false?

Reply:
Since activity feed has its own internal scrollbar, not at the sideabar content level